modules/swagger-parser/src/test/java/io/swagger/parser/SwaggerParserTest.java

```diff
@@ -800,13 +802,65 @@ public void testBadFormat() throws Exception {
         queryParameter = (QueryParameter) parameter;
         assertEquals(queryParameter.getName(), "query-param-collection-format-and-uniqueItems");
         assertEquals(queryParameter.getCollectionFormat(), "multi");
-        assertEquals(queryParameter.isUniqueItems(), true);
+        assertTrue(queryParameter.isUniqueItems());
+    }
+
+    @Test
+    public void testNumberAttributes() throws Exception {
```

Comment:
this test is from PR #444 - I include it in order to fix the relative path used in #444